adoc/admin-gpus.adoc

```diff
@@ -181,3 +181,143 @@ Check the kubelet log on the worker node that has a GPU. This may indicate error
+
+
+[source,bash]
+----
+$ cat <<EOF | kubectl apply -f -
+---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: nvidia.dcgm
+spec:
+  privileged: false
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  runAsUser:
+    rule: RunAsAny
+  fsGroup:
+    rule: RunAsAny
+  allowedHostPaths:
+    - pathPrefix: /var/lib/kubelet/pod-resources
+  volumes:
+  - hostPath
+  - configMap
+  - secret
+  - emptyDir
+  - downwardAPI
+  - projected
+  - persistentVolumeClaim
+  - nfs
+  - rbd
+  - cephFS
+  - glusterfs
+  - fc
+  - iscsi
+  - cinder
+  - gcePersistentDisk
+  - awsElasticBlockStore
+  - azureDisk
+  - azureFile
+  - vsphereVolume
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: nvidia:dcgm
+rules:
+- apiGroups:
+  - policy
+  resources:
+  - podsecuritypolicies
+  verbs:
+  - use
+  resourceNames:
+  - nvidia.dcgm
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: nvidia:dcgm
+roleRef:
+  kind: ClusterRole
+  name: nvidia:dcgm
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: Group
+  name: system:serviceaccounts:dcgm
+---
+----
+
+=== Create the DCGM Exporter
+
+The DCGM exporter monitors GPUs on each worker node and exposes metrics that can be queried.
+
+[source,bash]
+----
+$ kubectl create namespace dcgm
+$ kubectl create --namespace dcgm -f https://raw.githubusercontent.com/NVIDIA/gpu-monitoring-tools/master/dcgm-exporter.yaml
+----
+
+Check that the metrics are being collected:
+
+[source,bash]
+----
+$ NAME=$(kubectl get pods --namespace dcgm -l "app.kubernetes.io/name=dcgm-exporter" -o "jsonpath={ .items[0].metadata.name}")
+$ kubectl port-forward $NAME 8080:9400 &
+$ curl http://127.0.0.1:8080/metrics
+----
+
+=== Configure Prometheus
+
+After deploying Prometheus as explained in link:{docurl}/single-html/caasp-admin/#_monitoring[Monitoring], configure Prometheus to monitor the DCGM pods. Gather the cluster IPs of the pods to monitor:
+
+[source,bash]
+----
+$ kubectl get pods --namespace dcgm -l "app.kubernetes.io/name=dcgm-exporter" -o "jsonpath={ .items[*].status.podIP}"
+10.244.1.10 10.244.2.68
+----
+
+Add the DCGM pods to Prometheus's scrape configuration. Edit the Prometheus configmap:
+
+[source,bash]
+----
+$ kubectl edit --namespace monitoring configmap prometheus-server
+----
+
+Under the ``scrape_configs`` section add a new job:
+
+[source,yaml]
+----
+scrape_configs:
+...
+- job_name: dcgm
+  static_configs:
+  - targets: ['10.244.1.10:9400', '10.244.2.68:9400']
+...
+----
+
+Prometheus will automatically reload the new configuration.
+
+=== Add the Grafana Dashboard
+
+Import the DCGM Exporter dashboard into Grafana.
+
+In the Grafana web interface, navigate to Manage Dashboards and click `Import`. In the field "Import via grafana.com", enter the dashboard ID `12219`, and click `Load`.
```

Comment:
Should "Manage Dashboards" be formatted specially some way?  I'm not sure offhand how we refer to menu items in our docs.  The button name and menu labels are quoted in the next paragraph, so maybe this should also be quoted?

Reply:
Will await input from the docs team.

Reply:
You can use `menu:File[Manage Dashboards]`. I'll go ahead and change that during the review. :)